contracts/MetadataViews.cdc

```diff
@@ -737,4 +737,179 @@ pub contract MetadataViews {
         return Traits(traits)
     }
 
+    /// A struct to expose license information according to https://forum.onflow.org/t/flow-nft-license-project/4716
+    /// By default, there are only 12 possible license combinations,
+    /// so these are enabled by the 12 utility methods below the struct definition
+    /// to allow easy construction of each possible combination
+    /// For a given NFT, returning a license view is as simple as
+    /// calling the method for the desired view, like so:
+    ///
+    /*   pub fun resolveView(_ view: Type): AnyStruct? {
+            switch view {
+                case Type<MetadataViews.NFTLicense>():
+                    return MetadataViews.nlpVoteCom()
+                case ...
+    */
+    ///
+    /// The first version of the NFT license standard is only deployed
+    /// with the license names and text descriptions. Projects must refer
+    /// to the proposal and license documentation for specific descriptions of
+    /// the rights that each license grants.
+    ///
+    pub struct NFTLicense {
```

Comment:
Not a blocker at all or in scope for this PR, but I wonder how often comparisons would be utilized between `NFTLicense` structs.

It could be helpful to have a comparator `equals(_ other: NFTLicense): Bool` method conditioned on the equivalence of `licenses` arrays between views or some other similar equivalency (not completely sure of the business/legal logic here). Thinking about a situation where a platform/marketplace wants to ensure an NFT has the appropriate licensing rights for the use case.

Reply:
Good idea. I added it and a test